examples/validation/src/custom_handlers.py

```diff
@@ -0,0 +1,13 @@
+from custom_handlers_schema import CHILD_SCHEMA, PARENT_SCHEMA
+
+from aws_lambda_powertools.utilities.typing import LambdaContext
+from aws_lambda_powertools.utilities.validation import validator
+
+
+def get_child_schema(uri):
```

Comment:
nitpick: can we type `uri` parameter so customers know what they're dealing with?

Reply:
great catch.